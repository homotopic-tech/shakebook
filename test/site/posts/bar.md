---
title: The Bar in the mountains
date: 2020-06-01
tags: [bar, spatula]
---

Lorem ipsum
