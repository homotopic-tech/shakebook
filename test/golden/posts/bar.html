<html>
  <head>
    <title>The Bar in the mountains</title>
  </head>
  <body>
    <h1>This post is called The Bar in the mountains</h1>
    <p>
      <small>Posted on Monday, June 01, 2020 with tags:
         <a href="/posts/tags/bar">bar</a>
         <a href="/posts/tags/spatula">spatula</a>
      </small> 
    <p>It was generated from the source test/site/posts/bar.md</p>
    <p>For the url /posts/bar.html</p>
    <p><p>Lorem ipsum</p></p>
    <p>Here are the recent posts again:
      <ul>
        <li>
          <a href="/posts/quuz.html">The Quux In The Maw</a>
          <small>Posted on Monday, July 20, 2020 with tags:
              <a href="/posts/tags/quux">quux</a>
              <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
        <li>
          <a href="/posts/baz.html">The Baz In the Hills</a>
          <small>Posted on Sunday, June 28, 2020 with tags:
              <a href="/posts/tags/baz">baz</a>
              <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
        <li>
          <a href="/posts/bar.html">The Bar in the mountains</a>
          <small>Posted on Monday, June 01, 2020 with tags:
              <a href="/posts/tags/bar">bar</a>
              <a href="/posts/tags/spatula">spatula</a>
          </small>
        </li>
      </ul>
    </p>
  </body>
</html>
